### ISSUE TITLE
Point -> String representation incorrect?

### ISSUE THREAD

**Issue by vespakoen** [creof/doctrine2-spatial#32]
Hey!

I used your package to convert from binary to GeoJSON and back again, working great so far!

However, there was one line I had to modify in the core, so I guess it's a bug, perhaps I am just using the library incorrectly since I only use a couple of things from it, you decide =).

https://github.com/djlambert/doctrine2-spatial/blob/master/lib/CrEOF/Spatial/PHP/Types/AbstractGeometry.php#L193-L201

This returns something like `5,23523 50,1242124`

The `,` needed to be a `.` to make my DB stop returning errors when I tried to jam those coordinates in there, so I did a simple str_replace there (should have probably changed the toString method in the Point or something, but hey, this worked for my scenario.

Perhaps the package could support more methods like toGeoJSON and toSql, here is my very basic, initial code for doing that.

https://gist.github.com/vespakoen/9931066

**Comment by sadortun** [comment]
Hi @vespakoen ,

Can you `var_dump` your input data ?  The only way that code might return this would be if :

```php
$a = [1 , '2,4' ];
echo vsprintf('%s %s' , $a);
```

Thanks

**Comment by vespakoen** [comment]
Hey man! Thanks for coming back to this, the project where I needed this for has now been ported to Node.JS so it does not affect me anymore.

Unfortunately I have lost the code where I used it, so I cannot check if I used strings.

It might have already been fixed (or not even an issue) and since this issue is > 1 year old I guess it is safe to close this.

Anyways good luck with the project and thanks for making it!
